mathics/core/parser/tokeniser.py

```diff
@@ -230,7 +233,7 @@
     '<': ['Get', 'StringJoin', 'LessEqual', 'Less'],
     '=': ['SameQ', 'UnsameQ', 'Equal', 'Unset', 'Set'],
     '>': ['PutAppend', 'Put', 'GreaterEqual', 'Greater'],
-    '?': ['PatternTest'],
+    '?': ['Information', 'Definition','PatternTest'],
```

Comment:
OK, but then how we should implement it? I tried to imitate the behavior of @@ and >>, in a way that the tokeniser first look at the double operator, and afterward to the simple one.  This is why the change at line 236. What I do not realise is how and when the tokeniser distinguishes a prefix operator from an infix binary one...